suite/tests/CMakeLists.txt

```diff
@@ -3742,6 +3742,46 @@ foreach (run ${run_list})
   endif (enabled)
 endforeach(run)
 
+if (NOT DEBUG AND UNIX AND X64 AND NOT CMAKE_COMPILER_IS_CLANG)
+  set_tests_properties(
+    code_api|tool.drcacheoff.simple
```

Comment:
It is worth a comment explaining why we have a subset, and why we focus on drcachesim.

Reply:
Done.